cmd/mattermost/commands/user.go

```diff
@@ -398,6 +408,126 @@ func usersToBots(args []string, a *app.App) {
 	}
 }
 
+func updateBotUserPassword(command *cobra.Command, a *app.App, user *model.User) error {
+	password, err := command.Flags().GetString("password")
```

Comment:
I would expect this checks before any action is executed. Here we are checking if the password is blank after updating the user data, I think that is wrong, I don't mind to have it in more than one step, but concrete step is way more likely to fail than a database write.

Reply:
My point is, we need to move this checks out of the function, and put before any change is done in the database.